content-repo/extra-docs/releases/20.9.0.md

```diff
@@ -0,0 +1,408 @@
+- Improved Error handling messages.
+- Parameter hostname is now mandatory.
+
+#### Playbooks
+##### Endace Search Archive and Download  
+This playbook is deprecated. Use Endace Search Archive Download PCAP instead.
+
+##### Deprecated: Endace Search Archive Download PCAP  
+This playbook is deprecated. Use Endace Search Archive Download PCAP v2 instead.
+
+##### Endace Search Archive Download PCAP v2  
+Added support for directionless ip and port search, user friendly timeframe values, updated input and output variables to the playbook and their definitions.
+
+---
+
+### Expanse Pack v1.1.1 (Partner Supported)
+#### Integrations
+##### Expanse  
+Updated the version number in the user-agent header
+
+---
+
+### GDPR Pack v1.0.5
+#### Layouts
+- **layout-edit-GDPR_Data_Breach.json**
+- **layout-details-GDPR_Data_Breach.json**
+
+---
+
+### GitHub Pack v1.1.1
+#### Integrations
+##### GitHub  
+Improved implementation of parameters errors handling.
+
+---
+
+### Gmail Pack v1.0.5
+#### Integrations
+##### Gmail  
+Breaking Change: Removed the deprecated *event* argument from the ***gmail-list-users*** command.
+
+
+---
+
+### IBM QRadar Pack v1.0.8
+#### Integrations
+##### IBM QRadar  
+Added a parameter that enables you to specify the number of addresses to enrich per API call.
+
+---
+
+### IronDefense Pack v1.1.1 (Partner Supported)
+#### Classifiers
+##### IronDefense  
+Added new Classifier
+
+##### IronDefense - Incoming Mapper  
+Added new incoming mapper
+
+##### IronDefense - Classifier  
+Added new Classifier
+
+#### Integrations
+##### IronDefense  
+- Added ability to retrieve IronDefense alerts and events.
+- Improving descriptions.
+
+#### Layouts
+- **layout-details-IronDefense_IronDome_Notification.json**
+- **layout-details-IronDefense_Event_Notification.json**
+- **layout-details-IronDefense_Alert_Notification.json**
+
+---
+
+### Malware Pack v1.2.3
+#### Layouts
+- **layout-edit-Malware.json**
+- **layout-details-Malware.json**
+
+---
+
+### Malwarebytes Pack v1.0.2 (Partner Supported)
+#### Integrations
+##### Malwarebytes  
+Added Code for Usage Analytics.
+
+---
+
+### McAfee ESM v10 and v11 Pack v1.0.3
+#### Integrations
+##### Deprecated: McAfee ESM v10 and v11  
+Deprecated - We recommend using McAfee ESM v2 instead.
+
+---
+
+### Microsoft Graph User Pack v1.2.0
+#### Integrations
+##### Microsoft Graph User  
+Fixed an issue where the ***msgraph-user-list*** command paging argument did not work as expected.
+
+
+---
+
+### Palo Alto Networks Cortex XDR - Investigation and Response Pack v2.3.3
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response  
+Added default classifier and mapper.
+
+---
+
+### PassiveTotal Pack v1.0.1
+#### Integrations
+##### PassiveTotal  
+Added a limit to the number of related domains showed in the war room for enrichment commands.
+
+---
+
+### Phishing Pack v1.10.2
+#### Layouts
+- **layout-quickView-Phishing.json**
```

Comment:
What changed?

Reply:
Need to investigate why it happens on layouts only.